### DISCUSSION TITLE
Stitched Mesh with nodes not at the exact same locations.

### DISCUSSION THREAD

**Discussion by Fred-88** [idaholab/moose#25579] [Q&A Meshing] [answered]
Hi all,
I am using the StitchedMeshGenerator to connect two distinct mesh together.
It seems that this functionality only merges nodes that are at the exact same geometric location.
Does anyone knows if it possible to merge nodes from the stitch_boundaries_pairs that are not exactly at the same location; e.g. using a maximum distance parameter ?
Cordially
Fred

**Comment by GiudGiud** [comment, marked as answer]
Hello 

the MeshRepairGenerato will let you merge overlapping nodes with a tolerance that you can set to be big if you want

at your own risk though

**Reply by Fred-88** [reply]
Merci Guillaume, I had missed this functionality but will try in now.

**Reply by Fred-88** [reply]
Hi,
I have tried the MeshRepairGenerator and it works very well on my simple model; but I can see that it can be tricky in some cases as it applies to the whole mesh.
Thanks again.
Fred
